### DISCUSSION TITLE
Migrating hawtio from 2.17.7 to 3.0.1 - Activemq plug in (UI is missing)

### DISCUSSION THREAD

**Discussion by venkikeesara** [hawtio/hawtio#3241] [Q&A]
Hi ,
we recently migrated ActiveMQ from 5.16.7 to 5.18.3 . As part of the upgrade process , We had one  issue- https://issues.apache.org/jira/browse/AMQ-9300  and this is fixed  now. 
Now we migrated hawtio from 2.17.7 to 3.0.1 , but activeMQ UI is missing , no errors on browser side . Any help is appreciated 

Note - 
I verified both versions of code , looks like hawtio-war-2.17.7/img/icons/activemq is missing in 3.0.1 version . D owe have any solution

**Comment by venkikeesara** [comment]
@atoulme @olamy @jstrachan @sullis  - any thoughts

**Comment by tadayosi** [comment]
If you are talking about ActiveMQ Classic, we have no plan to support it, no plan to migrate the plugin to the new Hawtio. For ActiveMQ Artemis, the Artemis team is now actively working on to develop a plugin to support Hawtio with the latest Artemis broker. You'll be able to use it soon.

That doesn't exclude contributions from community. If you really want to make it work with the new Hawtio, we are more than happy to accept your contribution.

**Comment by venkikeesara** [comment]
Thank you @tadayosi  for your reply

**Comment by grgrzybek** [comment]
Related issues:
* #3534
* https://issues.apache.org/jira/browse/ARTEMIS-5334
